-- Found ament_cmake: 1.3.6 (/opt/ros/humble/share/ament_cmake/cmake)
-- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
-- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
-- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
-- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
  execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
  my_robot_interfaces --arguments-file msg/HardwareStatus.msg --output-dir
  /home/sambhav/ros2_ws/build/my_robot_interfaces/rosidl_adapter/my_robot_interfaces
  --output-file
  /home/sambhav/ros2_ws/build/my_robot_interfaces/rosidl_adapter/.idls)
  returned error code 1:

  Traceback (most recent call last):

    File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
      return _run_code(code, main_globals, None,
    File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
      exec(code, run_globals)
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
      sys.exit(main())
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 45, in main
      with open(args.arguments_file, 'r') as h:

  FileNotFoundError: [Errno 2] No such file or directory:
  'msg/HardwareStatus.msg'

Call Stack (most recent call first):
  CMakeLists.txt:17 (rosidl_adapt_interfaces)

[0m
-- Configuring incomplete, errors occurred!
See also "/home/sambhav/ros2_ws/build/my_robot_interfaces/CMakeFiles/CMakeOutput.log".
